packages/components/src/components/link-tile/__snapshots__/link-tile-styles.spec.ts.snap

```diff
@@ -17,40 +17,41 @@ exports[`getComponentCss() should return correct css for aspectRatio: { base: '1
 }, align: top, isCompact: { base: true, xs: false, s: true, m: false, l: true, xl: false } and hasGradient: true 1`] = `
 ":host {
   display: block;
-  height: fit-content !important;
+  height: fit-content;
 }
-:host ::slotted(picture), :host ::slotted(img) {
```

Comment:
These old selectors didn't work at all, right?

Reply:
To be honest, I don't know why they were generated (and how). Pure `::slotted(img)` should be enough, right?